data/brands/amenity/fast_food.json

```diff
@@ -9240,11 +9258,28 @@
         "cuisine": "beef_bowl",
         "name": "食其家",
         "name:en": "Sukiya",
-        "name:ja": "食其家",
+        "name:ja": "すき家",
         "name:zh": "食其家",
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Sukiya",
+      "locationSet": {"include": ["hk"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "すき家 Sukiya",
+        "brand:en": "Sukiya",
+        "brand:ja": "すき家",
+        "brand:wikidata": "Q6137375",
+        "cuisine": "beef_bowl",
+        "name": "すき家 Sukiya",
+        "name:en": "Sukiya",
+        "name:ja": "すき家",
+        "int_name:zh": "食其家",
```

Comment:
<!--
```suggestion
        "name:zh": "食其家",
```
-->
A bit curious about this, why not use "name:zh" but add a "int_" prefix 😀

Reply:
They use "食其家" in Taiwan. I considered not using it as "Sukiya" is the most common name used here too, but at least some people in HK know and uses this word. A few OSM users have been adding "name:zh=食其家" which I have to point out every time. This is annoying. For other solutions, I don't think you want to write `not:name:zh=食其家` here. 
However, `int_name:zh=` creates a problem on what to do about `brand:zh=` as well, so I'm fine with dropping it.

Reply:
Another resaon is `int_name` may cause problem in other data user, because they use `name:*` to display different language's name.

But you won't feel annoy from today, because once it added to NSI and publish a new version, every time you open iD//RapiD it will check the element's tag with documented in this repository. 😀

So `name=食其家 Sukiya` will be auto suggested to `name=Sukiya` because later one is documented in this repository.